src/index.js

```diff
@@ -12,12 +12,16 @@ const parseLocale = (preset, object, isLocal) => {
   let l
   if (!preset) return L
   if (typeof preset === 'string') {
-    if (Ls[preset]) {
-      l = preset
+    const presetLower = preset.toLowerCase()
+    if (Ls[presetLower]) {
+      l = presetLower
     }
     if (object) {
-      Ls[preset] = object
-      l = preset
+      Ls[presetLower] = object
+      l = presetLower
+    }
+    if (!l && preset.split('-').length > 1) {
```

Comment:
maybe we can simplify the logic to 
const presetSplit = preset.split('-')
if (!l && presetSplit.length) {
  return parseLocale(presetSplit[0])
}

Reply:
Yes. I'll fix it!